attendance-portal/src/Student/pages/index.js

```diff
@@ -0,0 +1,4 @@
+import Dashboard from './Dashboard';
+import Details from '../../Components/Details'
```

Comment:
Why is `<Details />`  in a new folder known as Components. If you prefer keeping the components in a separate folder. then don't bring those components in pages but instead directly import it from components

Suggestions:

1. rename Components -> components
2. create `index.js` file to unify all components related